examples/wdio/jasmine/jasmine.spec.js

```diff
@@ -1,6 +1,6 @@
 describe('webdriver.io page', () => {
-    it('should have the right title', () => {
-        browser.url('https://webdriver.io')
-        expect(browser).toHaveTitle('WebdriverIO · Next-gen browser and mobile automation test framework for Node.js | WebdriverIO')
+    it('should have the right title', async () => {
+        await browser.url('https://webdriver.io')
+        expect(await browser).toHaveTitle('WebdriverIO · Next-gen browser and mobile automation test framework for Node.js | WebdriverIO')
```

Comment:
may be 
 `await expect(browser).toHaveTitle('WebdriverIO · Next-gen browser and mobile automation test framework for Node.js | WebdriverIO')`

Reply:
Good catch! Will fix with another commit.